src/util/EventFetcher.js

```diff
@@ -16,16 +52,14 @@ function fetchEvents(numOfEvents) {
     let eventData = [];
     let fields = [];
     // eslint-disable-next-line
-      fetch(`${SCE_GOOGLE_API_URL}/api/mailer/getCalendarEvents?calendarID=primary&numOfEvents=${numOfEvents}`)
+      fetch(`${SCE_GOOGLE_API_URL}/api/calendar/getCalendarEvents?calendarID=${EVENTS_CAL}&numOfEvents=${numOfEvents}`)
       .then(data => data.json())
       .then(data => {
         eventData = data.calendarEvents;
         fields = eventData.map(event => ({
           'name': `${calendar} ${event.summary}`,
-          'value': 
-                  `${info} \`${event.description || 'No Info'}\` 
-                    \n${pushPin} \`${event.location || 'No Location'}\` 
-                    \n${clock} \`${event.start.dateTime}\``,
+          'value': `${locationNull(event.location)}
```

Comment:
you could also do
```suggestion
          'value': `${formatEventInfo(event)}
```
to really get that abstraction where u just send in an event and then a function up top formats the rest of the string, `clock` and all